lib/src/over_react_test/common_component_util.dart

```diff
@@ -151,142 +146,144 @@ void expectCleanTestSurfaceAtEnd() {
 /// Common test for verifying that unconsumed props are forwarded as expected.
 ///
 /// > Typically not consumed standalone. Use [commonComponentTests] instead.
-void testPropForwarding(BuilderOnlyUiFactory factory, dynamic childrenFactory(), {
```

Comment:
It looks like this file is exported publicly:
https://github.com/Workiva/over_react_test/blob/0f9e813aba6c8cc88f3fc8aeaad2a22118400d01/lib/over_react_test.dart#L15
so can we just comment out the body so that these functions don't get removed from the public API?

Reply:
Actually, now that I think about it, I wonder if it would make more sense to have a major release in which these are removed (or updated to throw `UnimplementedError` or something), and then re-add them once they're supported... I'd hate to break functionality depending on something like `getComponentPropKeys` without the proper versioning.

I'm on the fence; what are your thoughts?

Reply:
I was thinking that we'd have to release a new breaking version. Both `getComponentPropKeys` and `testPropForwarding` are publicly exported. In addition, the functionality of `commonComponentTests` has changed (no longer tests prop forwarding).